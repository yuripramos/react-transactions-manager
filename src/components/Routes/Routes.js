import React, { Component } from "react";
import { Switch, Route } from "react-router-dom";

import NotFound from "../common/NotFound";
import Content from "../Content";

import Layout from "../common/Layout";

class Routes extends Component {
  render() {
    return (
      <Switch>
        <Layout>
          <Route>
            <Switch>
              <Route exact path="/" component={Content} />
              <Route component={NotFound} />
            </Switch>
          </Route>
        </Layout>
      </Switch>
    );
  }
}

export default Routes;
