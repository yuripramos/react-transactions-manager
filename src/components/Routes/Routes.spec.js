import React from "react";
import { shallow } from "enzyme";

import Routes from "./Routes";

describe("Routes component", () => {
  it("should match snapshot", () => {
    expect(shallow(<Routes />)).toMatchSnapshot();
  });
});
