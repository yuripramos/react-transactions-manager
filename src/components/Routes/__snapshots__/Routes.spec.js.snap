// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes component should match snapshot 1`] = `
<Switch>
  <Layout>
    <Route>
      <Switch>
        <Route
          component={[Function]}
          exact={true}
          path="/nyttop"
        />
        <Route
          exact={true}
          path="/science/hot"
          render={[Function]}
        />
        <Route
          path="/science/hot/:idArticle"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/science/all"
          render={[Function]}
        />
        <Route
          path="/science/all/:idArticle"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/technology/hot"
          render={[Function]}
        />
        <Route
          path="/technology/hot/:idArticle"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/technology/all"
          render={[Function]}
        />
        <Route
          path="/technology/all/:idArticle"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/health/hot"
          render={[Function]}
        />
        <Route
          path="/health/hot/:idArticle"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/health/all"
          render={[Function]}
        />
        <Route
          path="/health/all/:idArticle"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/politics/hot"
          render={[Function]}
        />
        <Route
          path="/politics/hot/:idArticle"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/politics/all"
          render={[Function]}
        />
        <Route
          path="/politics/all/:idArticle"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/world/hot"
          render={[Function]}
        />
        <Route
          path="/world/hot/:idArticle"
          render={[Function]}
        />
        <Route
          exact={true}
          path="/world/all"
          render={[Function]}
        />
        <Route
          path="/world/all/:idArticle"
          render={[Function]}
        />
        <Route
          component={[Function]}
        />
      </Switch>
    </Route>
  </Layout>
</Switch>
`;
